<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configwindow</class>
 <widget class="QWidget" name="configwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>528</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>142</red>
        <green>142</green>
        <blue>142</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>142</red>
        <green>142</green>
        <blue>142</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>250</red>
        <green>250</green>
        <blue>250</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FAFAFA;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#EEEEEE</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QListWidget" name="m_widgetConfigTypes">
        <property name="styleSheet">
         <string notr="true">QListWidget::item{color:#7B7B7B;}
QListWidget::item:selected{background-color:#50aefa;color:#ffffff;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="gridSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>General</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Demo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Map</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Server</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stats</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="m_swSettings">
     <property name="styleSheet">
      <string notr="true">color:#909090;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_pageGeneral">
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>321</width>
         <height>114</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="m_lbGeneralLanguage">
          <property name="text">
           <string>Language</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="m_lbGeneralCheckForUpdates">
          <property name="text">
           <string>Check for updates</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="m_pbGeneralCheckForUpdates">
          <property name="styleSheet">
           <string notr="true">QPushButton{
text-align:left;
border:none;
margin-bottom:5px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../data/ressources.qrc">
            <normaloff>:/img/ui_elements/switch_off.png</normaloff>
            <normalon>:/img/ui_elements/switch_on.png</normalon>:/img/ui_elements/switch_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="m_lbGeneralAutoRefreshTranslations">
          <property name="text">
           <string>Auto refresh translations</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="m_pbGeneralAutoRefreshTranslations">
          <property name="styleSheet">
           <string notr="true">QPushButton{
text-align:left;
border:none;
margin-bottom:5px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../data/ressources.qrc">
            <normaloff>:/img/ui_elements/switch_off.png</normaloff>
            <normalon>:/img/ui_elements/switch_on.png</normalon>:/img/ui_elements/switch_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="m_lbGeneralAutoExpandMenu">
          <property name="text">
           <string>Expand main menu on startup</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="m_pbGeneralAutoExpandMenu">
          <property name="styleSheet">
           <string notr="true">QPushButton{
text-align:left;
border:none;
margin-bottom:5px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../data/ressources.qrc">
            <normaloff>:/img/ui_elements/switch_off.png</normaloff>
            <normalon>:/img/ui_elements/switch_on.png</normalon>:/img/ui_elements/switch_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="m_cbGeneralLanguage">
          <item>
           <property name="text">
            <string>English</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="m_pageDemo"/>
     <widget class="QWidget" name="m_pageMap"/>
     <widget class="QWidget" name="m_pageServer"/>
     <widget class="QWidget" name="m_pageStats">
      <widget class="QPushButton" name="m_pbStatsUseDefaultMasters">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>172</y>
         <width>36</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
text-align:left;
border:none;
margin-bottom:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../data/ressources.qrc">
         <normaloff>:/img/ui_elements/switch_off.png</normaloff>
         <normalon>:/img/ui_elements/switch_on.png</normalon>:/img/ui_elements/switch_off.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="m_lbStatsUseDefaultMasters">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Use default masterservers</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../data/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
